### DISCUSSION TITLE
Chat commands

### DISCUSSION THREAD

**Discussion by jhart326** [YimMenu/YimMenu#1970] [Q&A]
So I got a question on how to use the chat commands and what are the list of commands can the player do since you can give people chat command permissions in network player database if anyone knows how to use them please let me know
![Screenshot 2023-08-14 180132](https://github.com/YimMenu/YimMenu/assets/89509549/76e1dc33-fcf0-4c51-ae65-3add4a61d90f)

**Comment by pierrelasse** [comment]
The entire chat command system is a mystery  to me

**Comment by Rabtos** [comment]
Would it be possible with this to change other players vehicle handling? I mean if they do the command themself for their car?

**Reply by Yimura** [reply]
No it’s not possible, in the end commands are still executed on your instance on the game. Meaning you’re still limited and can’t modify things in their game. Unless they themselves install YimMenu and apply the handling modifications themselves.

**Comment by ghost** [comment]
I wanna make myself useful and make proper docs for all the chat commands, but also I would need to play around with it first to understand how everything works. Also idk i think im just dumb
